eleventy.config.js

```diff
@@ -56,6 +56,15 @@ module.exports = function configure(eleventyConfig) {
     });
   });
 
+  eleventyConfig.addFilter('mediaGetFullURL', (allMedia, featuredMediaId) => {
+    const media = allMedia.find((item) => {
+      return featuredMediaId === item.id;
+    });
+
+    // What could possibly go wrong? :/
+    return media ? media.media_details.sizes.full.source_url : '';
```

Comment:
This is a bit YDD (YOLO Driven Development). Could you be more defensive or is this data structure guaranteed?

Reply:
> This is a bit YDD (YOLO Driven Development). Could you be more defensive or is this data structure guaranteed?

This is the structure we get from the API. I'll add tests and some checks.

Reply:
I added a helper function to get the size of a "media" and it's covered by tests.